feedback.md

```diff
@@ -0,0 +1,32 @@
+Hi Kim!
+
+Many thanks for this awesome submission. I went through all (I hope) files and have a few follow-up questions - no coding needed, just conceptual stuff!
+
+Hope that's fine! :)
```

Comment:
Really liked reading it as well! Super clear, well documented, well tested :) Great job!

Reply:
I'm also curious about the `arbetsfoermedlingen` references, haha..

Reply:
Haha yeah saw that one too 😂

Reply:
I thought it was fitting for a project with a focus on workers :D

Reply:
… and then I changed it to amuse myself and keep a theme, hence santa claus, I was gonna make a list of children and have the workers calculate if they're naughty or not.

Reply:
… and then I realized it was a little bit of a sidetrack so I just focused on the task ❗️